public/index.html

```diff
@@ -24,7 +24,13 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
-    <title>Casamico</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@600;400&display=swap" rel="stylesheet">
+    <style>
+      @import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@600;400&display=swap');
+    </style>
+    <title>React App</title>
```

Comment:
Merge conflicts are tough. You can see in the Pull Request here that I changed the title from `React App` to `Casamico`. Could you revert this change (just manually change the title again)?

p.s. this kind of thing is super easy to miss during merge conflicts so don't worry about it